core/runtime/tests/mod.rs

```diff
@@ -82,6 +83,7 @@ fn setup(mode: Mode) -> (JsRuntime, Arc<AtomicUsize>) {
     get_error_class_fn: Some(&|error| {
       crate::error::get_custom_error_class(error).unwrap()
     }),
+    shared_array_buffer_store: Some(CrossIsolateStore::default()),
```

Comment:
@bartlomieju @lucacasonato haven't found any sample where `shared_array_buffer_store` is set, but this did the trick.

The test fails on `main` (when `shared_array_buffer_store` is set)

Reply:
That's correct!